SharpPcap/LibPcap/LibPcapLiveDevice.cs

```diff
@@ -189,6 +189,39 @@ public override void Open(DeviceMode mode, int read_timeout, MonitorMode monitor
             Open(mode, read_timeout, monitor_mode, 0);
         }
 
+        /// <summary>
+        /// Open the device. To start capturing call the 'StartCapture' function
+        /// </summary>
+        /// <param name="flags"></param>
+        /// <param name="readTimeoutMilliseconds"></param>
+        /// <param name="remoteAuthentication"></param>
+        public void Open(OpenFlags flags,
+                 int readTimeoutMilliseconds,
+                 RemoteAuthentication remoteAuthentication)
```

Comment:
```suggestion
                 ICredentials remoteAuthentication)
```

Reply:
Is it really necessary to provide the credentials again?
`PcapInterface` already cache the credentials that were used during `pcap_findalldevs_ex`

See https://github.com/chmorgan/sharppcap/blob/master/SharpPcap/LibPcap/PcapInterface.cs#L70